### DISCUSSION TITLE
BONSAI, Support for saving freestyle lines into shading_style.json -> Make shading styles for hidden lines

### DISCUSSION THREAD

**Discussion by Slackking1** [IfcOpenShell/IfcOpenShell#5521] [Ideas]
It's would be really neat if together with the rest of the scene settings, freestyle lines could be saved into the configuration as well. 
Maybe i'm missing another smart way of doing it, but it seems freestyle lines and as an underlay is the way to go for hidden lines. But configuration is lost for the lines between sessions.

**Comment by theoryshaw** [comment]
+1
related: https://github.com/IfcOpenShell/IfcOpenShell/issues/5330

**Reply by Slackking1** [reply]
Totally agree, it's a lot of clicking in each new session.

**Comment by Slackking1** [comment]
On another note, even better would be having it supported as a type of linework. When using underlay for hidden lines, since it's a raster, printed sheets/drawings get big in size, and the graphics don't scale.
